_maps/configs/SYN-C_courage-VOID.json

```diff
@@ -0,0 +1,19 @@
+{
+	"map_name": "Courage-class Strike Dropship",
+	"map_short_name": "Courage-class",
+	"prefix": "SYN-C",
```

Comment:
“prefix”: “SSV”,
“namelists”: [“SPACE”, “WEAPONS”, “NATURAL_AGGRESSIVE”, “BEASTS”]

Prefix should follow established schemes for factions (SSV for syndicate), namelists suggested based on other syndicate ships